src/anomalib/models/components/base/anomaly_module.py

```diff
@@ -241,4 +242,9 @@ def load_state_dict(self, state_dict: OrderedDict[str, Tensor], strict: bool = T
         """
         # Used to load missing normalization and threshold parameters
         self._load_normalization_class(state_dict)
+        # Used to load fpr_limit in AURPO before create_metric_collection
+        if not hasattr(self, "pixel_metrics") and "pixel_metrics.AUPRO.fpr_limit" in state_dict.keys():
```

Comment:
Please put it in a function like `self._load_normalization_class`.

Reply:
Have fixed!